### PR TITLE
Automatically install correct bundler version (fixes #13)

### LINKED ISSUE #13: Automatically install correct bundler version
Bundler 2 has become very finicky and now requires always the exact bundler version to be installed as given in the Gemfile.lock.

The workaround at the moment is to add a
```
set :bundler_version, 'xxx'
```
to the capistrano config that matches the version given in `Gemfile.lock`.

It would be a bit nicer if capistrano-opscomplete would automatically install the bundler version given in the Gemfile.lock.

### PR DESCRIPTION
**Pull request by dastra-mak**
I tested the task manually for one of my private projects.

### CODE REVIEW

File: lib/capistrano/opscomplete/tasks.rake [hunk 1]
```diff
@@ -52,9 +52,11 @@ namespace :opscomplete do
     # desc 'Install bundler gem'
     task :install_bundler do
       on roles fetch(:rbenv_roles, :all) do
-        if fetch(:bundler_version, false)
+        specific_bundler_version = fetch(:bundler_version, false) || app_gemfile_bundled_with_version
```
Comment (kratob) [inline lib/capistrano/opscomplete/tasks.rake]: Das ist nur Nitpicking, aber naheliegender wäre `fetch(:bundler_version, app_gemfile_bundled_with_version)`.

**Review by kratob — COMMENTED** [review]
Für mich sieht das gut aus.

@makrobin magst du auch noch drauf schauen? Magst du das weiterhin lieber selbst releasen, oder sollen wir das einfach machen?

**Comment by kratob** [comment]
This would fix #13. It's technically a breaking change, since customers updating this gem will now automatically install the bundler version set in their project repo's Gemfile.lock, instead of keeping the bundler version that happens to be installed.

However, since that Bundler version obviously at least works locally and I do not think this is a big deal. It will however do the right thing by default for everyone using Bundler 2.

If someone manually specified a bundler version before it will still win.

**Review by makrobin — APPROVED** [review]
That looks cool to me, thanks for your contribution @dastra-mak <3.

@kratob feel free to do releases, but please let our Ops-Team know when you do so.  :)

**Comment by kratob** [comment]
@dastra-mak you can handle the release. I'll give you permissions for the gem.

**Pull request merged**